lib/assets/javascripts/cdb

```diff
@@ -1 +1 @@
-Subproject commit a2b8834622f2b4d3b849a7e810cff998d982844d
+Subproject commit 5bee7c8c6c6b2a62ec29c598f9c4331a9cb831fb
```

Comment:
This shouldn't have updated if you didn't make any changes to cartodb.js, which it doesn't look like you did.